cookbook/email/dev_environment.rst

```diff
@@ -119,6 +119,51 @@ the replaced address, so you can still see who it would have been sent to.
     These are ``X-Swift-Cc`` and ``X-Swift-Bcc`` for the ``CC`` and ``BCC``
     addresses respectively.
 
+Sending to a Specified Address, but with exceptions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Suppose you want to have all email sent to a specific address, instead
+of the address actually specified when sending the message (like in the above scenario),
+but you want certain email addresses not to be redirected in this way.
```

Comment:
This has become a very long sentence, let's try to make it a bit shorter. What do you think about?

````
Suppose you want all emails to be sent to ``dev@example.com``, except
from the emails sent to ``admin@specialdomain.com``. In this case, you
can use the ``delivery_whitelist`` option:

.. configuration-block::

    .. code-block:: yaml

        # app/config/config_dev.yml
        swiftmailer:
            delivery_address: dev@example.com
            delivery_whitelist:
                # all email addresses matching this regex will *not* be
                # redirected to dev@example.com
                - "/@mydomain.com$/"

                # all emails sent to admin@specialdomain.com won't
                # be redirected to dev@example.com too
                - "/^admin@specialdomain.com$/"

    .. code-block:: xml

        <?xml version="1.0" charset="UTF-8" ?>
        <container xmlns="http://symfony.com/schema/dic/services"
            xmlns:swiftmailer="http://symfony.com/schema/dic/swiftmailer"
        >
            <swiftmailer:config delivery-address="dev@example.com">
                <!-- all email adress matching this regex will *not* be redirected
                    to dev@example.com -->
                <swiftmailer:delivery-whitelist-pattern>/@mydomain.com$/</swiftmaler:delivery-whitelist-pattern>

                <!-- all emails sent to admin@specialdomain.com won't be
                    redirectd to dev@example.com too -->
                <swiftmailer:delivery-whitelist-pattern>/^admin@specialdomain.com$/</swiftmailer:delivery-whitelist-pattern>
            </swiftmailer:config>
````